packages/mobile/src/screens/chat-screen/ChatListScreen.tsx

```diff
@@ -143,7 +160,17 @@ export const ChatListScreen = () => {
       <ScreenContent>
         <View style={styles.shadow} />
         <View style={styles.rootContainer}>
-          {chatsStatus === Status.SUCCESS ? (
+          {isLoading ? (
+            Array(4)
```

Comment:
More nits:
You could do this more succinctly with the second arg to `Array.from`: 
```
Array.from(
  {length: 4},
  (_, index) => <ChatListItemSkeleton />
)
```
Also surprised that linting doesn't yell at you for using an index as key here. 
You could always just pull this out into a separate component that doesn't use a map

Reply:
Reason I didn't pull out to separate component is bc I wanted to re-use the same component for when we've already loaded some chats, and then scroll down and load more. So we use it in 2 places - on initial load (with fade-out), and on subsequent loads (no fade-out).
And yeah linting didn't complain for some reason. In this case tho I think it's fine bc we only ever use it once and the index will always be 0-4.